.github/workflows/update-templates-to-examples.yml

```diff
@@ -206,11 +206,11 @@ jobs:
           ref: ${{ github.event.pull_request.head.ref }}
       - name: Check-out fresh Starter template
         run: |
-          rm -rf examples/quickstart
-          mkdir -p examples/quickstart
-          printf 'info@zenml.io' | zenml init --path examples/quickstart --template starter --template-with-defaults
+          rm -rf examples/mlops_starter
```

Comment:
I still see only `examples/quickstart` in this branch + the tag is not modified above - is this intended? The tag shall also be updated in the CLI base.

Reply:
basically I want to write the old quickstart to mlops_starter and the new quickstart is fully written within this repo

Reply:
You'd better `git mv` it in this case 🙂 
Now I see some issues with `git diff` updating quickstart (old). I created folder manually and checking if this helps